src/tests/mock/instantiate.rs

```diff
@@ -0,0 +1,19 @@
+use cosmwasm_schema::cw_serde;
+use cosmwasm_std::{DepsMut, Env, MessageInfo, Response, StdResult};
+
+use crate::{bindings::query::ElysQuery, states::*, types::Order};
+
+#[cw_serde]
+pub struct InstantiateMockMsg {
+    pub epoch_cycle_interval: u128,
+    pub orders: Vec<Order>,
```

Comment:
there's no such thing as Order in this contract

Reply:
@politeWall 
Fixed